### PR TITLE
python3Packages.opuslib: init at 3.0.3

### PR DESCRIPTION
**Pull request by thelegy**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7318,6 +7318,12 @@
     githubId = 8547242;
     name = "Stefan Rohrbacher";
   };
+  "thelegy" = {
+    email = "mail+nixos@0jb.de";
+    github = "thelegy";
+    githubId = "3105057";
+    name = "Jan Beinke";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: this should be a separate commit
```
maintainers: add thelegy
```
```suggestion
  "thelegy" = {
    email = "mail+nixos@0jb.de";
    github = "thelegy";
    githubId = 3105057;
    name = "Jan Beinke";
  };
```

File: pkgs/development/python-modules/opuslib/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
```
Comment (jonringer) [inline pkgs/development/python-modules/opuslib/default.nix]: most packages will start on the first line
```suggestion
{ buildPythonPackage,
```

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -4629,6 +4629,8 @@ in {
 
   omegaconf = callPackage ../development/python-modules/omegaconf { };
 
+  opuslib = disabledIf (!isPy3k) (callPackage ../development/python-modules/opuslib { });
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix]: move this into the expresssion
```suggestion
  opuslib = callPackage ../development/python-modules/opuslib { };
```

File: pkgs/development/python-modules/opuslib/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  fetchFromGitHub,
+  libopus,
+  nose,
+  stdenv,
+  substituteAll,
+}:
+
+buildPythonPackage rec {
+  pname = "opuslib";
+  version = "3.0.3";
```
Comment (jonringer) [inline pkgs/development/python-modules/opuslib/default.nix]:
```suggestion
  version = "3.0.3";
  disabled = isPy27;
```

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (thelegy) [reply maintainers/maintainer-list.nix]: i splitted the commits and force pushed to this pr

File: pkgs/top-level/python-packages.nix (hunk 3 above)
Comment (thelegy) [reply pkgs/top-level/python-packages.nix]: thanks for the advice

File: pkgs/development/python-modules/opuslib/default.nix [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      src = ./opuslib-paths.patch;
+      opusLibPath = "${libopus}/lib/libopus${stdenv.hostPlatform.extensions.sharedLibrary}";
+    })
+  ];
+
+  checkInputs = [ nose ];
+
+  meta = with stdenv.lib; {
+    description = "Python bindings to the libopus, IETF low-delay audio codec";
+    license = licenses.bsd3;
```
Comment (jonringer) [inline pkgs/development/python-modules/opuslib/default.nix]: please add a homepage

Comment (thelegy) [reply pkgs/development/python-modules/opuslib/default.nix]: is the github repo an adequate homepage? I do not know any other.

Comment (jonringer) [reply pkgs/development/python-modules/opuslib/default.nix]: yes, that's fine

File: maintainers/maintainer-list.nix [hunk 6]
```diff
@@ -7318,6 +7318,12 @@
     githubId = 8547242;
     name = "Stefan Rohrbacher";
   };
+  "thelegy" = {
+    email = "mail+nixos@0jb.de";
+    github = "thelegy";
+    githubId = "3105057";
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]:
```suggestion
    githubId = 3105057;
```

Comment (jonringer) [reply maintainers/maintainer-list.nix]: this was in the original suggestion :)

Comment (thelegy) [reply maintainers/maintainer-list.nix]: sorry, i totally missed that

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
https://github.com/NixOS/nixpkgs/pull/83221
11 package marked as broken and skipped:
linuxPackages_hardkernel_4_14.prl-tools php72Packages-unit.php_excel php72Packages.php_excel php73Packages-unit.php_excel php73Packages.php_excel php74Packages-unit.couchbase php74Packages-unit.pcs php74Packages-unit.php_excel php74Packages.couchbase php74Packages.pcs php74Packages.php_excel

2 package built:
python37Packages.opuslib python38Packages.opuslib
```

**Comment by jonringer** [comment]
going to wait for ofborg to eval, and then merge

**Comment by jonringer** [comment]
@thelegy congrats on your first addition to nixpkgs :)

**Comment by thelegy** [comment]
thanks
